src/tests/suite/daos_rebuild_ec.c

```diff
@@ -871,7 +871,7 @@ rebuild_ec_parity_multi_group(void **state)
 }
 
 #define SNAP_CNT	20
-#define EC_CELL_SIZE	1048576
+#define EC_CELL_SIZE	DAOS_EC_CELL_DEF
```

Comment:
EC_CELL_SIZE is defined multiple times in this file, though not this patch issue.